tests/integration/labs/single.yaml

```diff
@@ -0,0 +1,124 @@
+---
+lab:
+  description: ""
+  notes: ""
+  title: cisco.iosxr.iosxr
+  version: 0.2.2
+links:
+  - id: l0
+    n1: n0
+    n2: n1
+    i1: i1
+    i2: i0
+    conditioning: {}
+    label: xr9kv-0-Mgmt/0<->ext-conn-0-port
+nodes:
+  - boot_disk_size: null
+    configuration: |2-
+      hostname iosxr
+      domain name ansible.com
+      username cisco
+       group root-lr
+       group sysadmin
+       group cisco-support
+       secret 10 $6$r9zWl1mjMw94Dl1.$V7Xh1CuqtHHEbfRjHqFjZlMnFrjCivkjbSYPoH/S6MjQBR.r/G3sFk3PQre.TsnSpfy.hKf.35Tclt8KkRzaQ.
+       password 7 05080F1C2243
+      !
+      aaa authentication login default local
+      cdp
+      line default
+       login authentication default
+       transport input ssh
+      !
+      call-home
+       service active
+       contact smart-licensing
+       profile CiscoTAC-1
+        active
+        destination transport-method email disable
+        destination transport-method http
+       !
+      !
+      control-plane
+       management-plane
+        inband
+         interface all
+          allow all
+          allow SSH
+         !
+        !
+       !
+      !
+      interface MgmtEth0/RP0/CPU0/0
+       cdp
+       ipv4 address dhcp
+      !
+      interface GigabitEthernet0/0/0/0
+       cdp
+       ipv4 address 192.168.1.1 255.255.255.0
```

Comment:
Can we reset the config under this interface?

Reply:
done!